src/include/access/relscan.h

```diff
@@ -127,6 +132,16 @@ typedef struct FileScanDescData
 	/* custom data formatter */
 	FormatterData *fs_formatter;
 	
+	/* formatter type and name */
```

Comment:
the indent of declaration need to be consistent with existing code

Reply:
I have updated my code due to your comments. thx!